### PR TITLE
Refactor the AWT Fonts PR

### PR DESCRIPTION
**Pull request by igordmn**
[Fix build](https://github.com/rock3r/skiko/commit/a02ffd075967ac21290d55d0e6cc63db82d55b45)
[Revert coroutines to 1.5.2](https://github.com/rock3r/skiko/commit/07ea5d9d7272d5676d5927f7bba8b3af0fbec727) (it seems we don't need the new version now, and we can reenable the tests)
[Fix font style conversions](https://github.com/rock3r/skiko/commit/d9b3a2ba17aef980cf77e1b5c2c5c28d603ed3bd)
[Replace println by Logger](https://github.com/rock3r/skiko/commit/5f3527a90eb8a768eb9e270526797366b053d288)
[Ignore failing font families](https://github.com/rock3r/skiko/commit/620b8e98e4e5156ba3725b8436f2069085903de3)
[Revert refactorings to simplify the PR](https://github.com/rock3r/skiko/commit/53f00c7d6be5ac0834ba4b4c3b9b4e3ea5cb0f10) (we can make a separate PR for them)
[Move font functionality to org.jetbrains.skiko.awt.font](https://github.com/rock3r/skiko/commit/cba4a3bcf6669fe538fd2686402ae276ea73c98e)
[Remove @InternalSkikoApi, mark internal instead](https://github.com/rock3r/skiko/commit/3a80116a8504aa75ccd3d87767dbf8c074f8c71b)

### CODE REVIEW

File: skiko/src/awtTest/kotlin/org/jetbrains/skiko/awt/font/InMemoryTypefaceCacheTest.kt [hunk 1]
```diff
@@ -40,7 +39,7 @@ class InMemoryTypefaceCacheTest {
             "The font we're trying to add must not already be loaded"
         )
 
-        val fontFile = createTempFile("awtfontmanagertest", "testfont")
+        val fontFile = kotlin.io.path.createTempFile("awtfontmanagertest", "testfont")
```
Comment (rock3r) [inline skiko/src/awtTest/kotlin/org/jetbrains/skiko/awt/font/InMemoryTypefaceCacheTest.kt]: Should we restore the import?

File: skiko/src/awtTest/kotlin/org/jetbrains/skiko/awt/font/SkiaFontProviderTest.kt [hunk 2]
```diff
@@ -0,0 +1,19 @@
+package org.jetbrains.skiko.awt.font
+
+import kotlinx.coroutines.runBlocking
+
+private fun Iterable<String>.ignoreVirtualAwtFontFamilies() =
```
Comment (rock3r) [inline skiko/src/awtTest/kotlin/org/jetbrains/skiko/awt/font/SkiaFontProviderTest.kt]: Are these unused now?

**Review by rock3r — COMMENTED** [review]
Only two specific notes; I don't understand some of the style changes (especially some KDoc changes that seem pejorative, and a number of empty lines at EOF were removed) but it's not a blocker

File: skiko/src/awtTest/kotlin/org/jetbrains/skiko/awt/font/SkiaFontProviderTest.kt (hunk 2 above)
Comment (igordmn) [reply skiko/src/awtTest/kotlin/org/jetbrains/skiko/awt/font/SkiaFontProviderTest.kt]: Wow, after moving the tests, I accedently delete 2 of them. Restored

File: skiko/src/awtTest/kotlin/org/jetbrains/skiko/awt/font/InMemoryTypefaceCacheTest.kt (hunk 1 above)
Comment (igordmn) [reply skiko/src/awtTest/kotlin/org/jetbrains/skiko/awt/font/InMemoryTypefaceCacheTest.kt]: Fixed

**Comment by igordmn** [comment]
> Only two specific notes; I don't understand some of the style

I just restored the previous state, before the PR, to exclude these refactorings from the change list (it is hard to distinguish refactorings and the feature in the big PR).

We can see that there are no changes other than the font changes in the PR if we compare with the base commit:
image

I plan to make separate PR for refactorings after the merge.

**Review by rock3r — APPROVED** [review]

**Pull request merged**